packages/calcite-components/src/components/flow-item/flow-item.e2e.ts

```diff
@@ -327,4 +331,21 @@ describe("calcite-flow-item", () => {
     expect(toggleSpy).toHaveReceivedEventTimes(1);
     expect(await panel.getProperty("closed")).toBe(true);
   });
+
+  it("should set embedded on slotted alerts", async () => {
```

Comment:
This is testing an internal prop. If screenshot tests cover this, we should drop this (and similar) tests.

Reply:
Screenshot tests do cover this, but I do think this adds some protection.